config.js

```diff
@@ -396,8 +396,8 @@ config.email = {
     'smtpTransport': {
         'service': 'Gmail',
         'auth': {
-            'user': 'my.email@gmail.com',
-            'pass': 'myemailpassword'
+            'user': 'taybson@gmail.com',
```

Comment:
Looks like you checked in your `config.js`, you should change your gmail password as it's now been shown to the world for a month.